ably/rest_client.go

```diff
@@ -66,21 +62,19 @@ func (c *RestClient) Time() (time.Time, error) {
 		return time.Time{}, err
 	}
 	if len(times) != 1 {
-		return time.Time{}, fmt.Errorf("Expected 1 timestamp, got %d", len(times))
+		return time.Time{}, fmt.Errorf("expected 1 timestamp, got %d", len(times))
```

Comment:
Why lower case "expected" whereas below you have "Unexpected" i.e. sentence case.  I'm not clear on why this commit was even made given it creates inconsistencies in case now?